ramls/patron-block-limit.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Patron block limits schema",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "A UUID identifying the limit",
+      "$ref": "raml-util/schemas/uuid.schema"
+    },
+    "patronGroupId": {
+      "type": "string",
+      "description": "A UUID identifying the patron group",
+      "$ref": "raml-util/schemas/uuid.schema"
+    },
+    "conditionId": {
+      "type": "string",
+      "description": "A UUID identifying the condition",
+      "$ref": "raml-util/schemas/uuid.schema"
+    },
+    "limit": {
```

Comment:
Maybe we should choose a different name for this field, e.g. `value`? In terms of this feature, a limit is an aggregate of referenced entities' IDs and the limit's **value**.

Reply:
renamed